selfdrive/manager/manager.py

```diff
@@ -169,6 +172,19 @@ def manager_thread() -> None:
     if shutdown:
       break
 
+def logTimestamps(sm, msgqs):
```

Comment:
this should be recoverable from the logs

Reply:
But this is were it is inserted in the logs (?). I tried to find this info in the logs without this function but could not find it.